deployment/cdk/opensearch-service-migration/lib/common-utilities.ts

```diff
@@ -78,6 +78,15 @@ export function parseAndMergeArgs(baseCommand: string, extraArgs?: string): stri
     return fullCommand;
 }
 
+export function createTargetPasswordAccessPolicy(targetPasswordSecretArn: string): PolicyStatement {
```

Comment:
nit: rename function to `getTargetPasswordAccessPolicy`, this isn't creating a resource.